tests/test_face_recognition.py

```diff
@@ -20,65 +20,121 @@
+        img = api.load_image_file(
+            os.path.join(
+                os.path.dirname(__file__),
+                'test_images',
+                'obama.jpg'))
         face_landmarks = api._raw_face_landmarks(img)
         example_landmark = face_landmarks[0].parts()[10]
 
-        assert len(face_landmarks) == 1
-        assert face_landmarks[0].num_parts == 68
-        assert (example_landmark.x, example_landmark.y) == (552, 399)
+        self.assertEqual(len(face_landmarks), 1)
+        self.assertEqual(face_landmarks[0].num_parts, 68)
+        self.assertEqual((example_landmark.x, example_landmark.y), (552, 399))
 
     def test_face_landmarks(self):
-        img = api.load_image_file(os.path.join(os.path.dirname(__file__), "test_images", "obama.jpg"))
+        img = api.load_image_file(
+            os.path.join(
+                os.path.dirname(__file__),
+                'test_images',
+                'obama.jpg'))
         face_landmarks = api.face_landmarks(img)
 
-        assert set(face_landmarks[0].keys()) == set(['chin', 'left_eyebrow', 'right_eyebrow', 'nose_bridge', 'nose_tip', 'left_eye', 'right_eye', 'top_lip', 'bottom_lip'])
-        assert face_landmarks[0]['chin'] == [(369, 220), (372, 254), (378, 289), (384, 322), (395, 353), (414, 382), (437, 407), (464, 424), (495, 428), (527, 420), (552, 399), (576, 372), (594, 344), (604, 314)]
+        self.assertEqual(
+            set(face_landmarks[0].keys()),
+            set(['chin', 'left_eyebrow', 'right_eyebrow', 'nose_bridge',
+                 'nose_tip', 'left_eye', 'right_eye', 'top_lip',
+                 'bottom_lip']))
+        self.assertEqual(
+            face_landmarks[0]['chin'],
+            [(369, 220), (372, 254), (378, 289), (384, 322), (395, 353),
+             (414, 382), (437, 407), (464, 424), (495, 428), (527, 420),
+             (552, 399), (576, 372), (594, 344), (604, 314)])
 
     def test_face_encodings(self):
-        img = api.load_image_file(os.path.join(os.path.dirname(__file__), "test_images", "obama.jpg"))
+        img = api.load_image_file(
+            os.path.join(
+                os.path.dirname(__file__),
+                'test_images',
+                'obama.jpg'))
         encodings = api.face_encodings(img)
 
-        assert len(encodings) == 1
-        assert len(encodings[0]) == 128
+        self.assertEqual(len(encodings), 1)
+        self.assertEqual(len(encodings[0]), 128)
 
     def test_compare_faces(self):
-        img_a1 = api.load_image_file(os.path.join(os.path.dirname(__file__), "test_images", "obama.jpg"))
-        img_a2 = api.load_image_file(os.path.join(os.path.dirname(__file__), "test_images", "obama2.jpg"))
-        img_a3 = api.load_image_file(os.path.join(os.path.dirname(__file__), "test_images", "obama3.jpg"))
-
-        img_b1 = api.load_image_file(os.path.join(os.path.dirname(__file__), "test_images", "biden.jpg"))
+        img_a1 = api.load_image_file(
```

Comment:
@ageitgey Yeah, I can change them back. They're split for PEP8 compatibility but you can define a new max-line-length in your Flake8 configuration